chem/chemics_init.F

```diff
@@ -677,13 +677,13 @@ subroutine chem_init (id,chem,emis_ant,scalar,dt,bioemdt,photdt,chemdt,stepbioe,
 !!! TUCCELLA
    if( .NOT. config_flags%restart ) then
    kpp_select: SELECT CASE(config_flags%chem_opt)
-     CASE (GOCARTRACM_KPP,RACM_KPP,RACMPM_KPP,RACMSORG_KPP,RACM_MIM_KPP,RACM_ESRLSORG_KPP, &
-           RACMSORG_AQCHEM_KPP,RACM_ESRLSORG_AQCHEM_KPP)
+CASE(GOCARTRACM_KPP,RACM_KPP,RACMPM_KPP,RACMSORG_KPP,RACM_MIM_KPP,RACM_ESRLSORG_KPP,&
```

Comment:
No reason to change this line, right?

Reply:
Yes, we are adding RACM_SOA_VBS_KPP and RACM_SOA_VBS_KPP_AQCHEM to the initialization of CO2, CH4, etc.

Reply:
Line 680 can be identical, there is no need to change it, the original indenting was correct

Reply:
Ok